utils.js

```diff
@@ -1,6 +1,12 @@
 const fs = require('fs');
 const path = require('path');
 
```

Comment:
should there be a [`.spectral.yml` file](https://docs.spectralops.io/usage/configuration/#spectralyaml)? or is there a common ReadMe ruleset package to import?

or mock the config file [like this](https://github.com/tylerlong/spectral-demo/blob/0508d28afd8a5d92d0b1e94a44dc0c4cdaa226b9/src/index.ts#L37-L44)?

Reply:
or look for a [config file in the repo](https://github.com/codacy/codacy-spectral/blob/7f69cc151629fe948b8b4436094d029e2ef58e80/src/engineImpl.ts#L165-L189)?

Reply:
We've moved linting to the backend for now, so this code can safely be removed.